ds_judgements_public_ui/sass/includes/_search.scss

```diff
@@ -112,10 +118,10 @@
   }
 
   &__header-text {
-    margin-left: 3rem;
+    line-height: 25px;
```

Comment:
Can we use the line-height variables here (as a note, px line heights will cause issues with accessibility)